service/src/test/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelperTest.java

```diff
@@ -22,11 +27,17 @@
 import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
 import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.unwrapElements;
 import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.buildCallout;
+import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.buildCalloutWithNextHearing;
 import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.buildSubjectLine;
 import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.buildSubjectLineWithHearingBookingDateSuffix;
 
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {FixedTimeConfiguration.class})
 class EmailNotificationHelperTest {
 
+    @Autowired
+    Time time;
```

Comment:
If you desperately want to use time in the main class then instead of what you have here you can do 
```java
private Time time = new FixedTimeConfiguration().stoppedTime();
```
and remove all of the annotations on top of the test and autowired annotation